Documentation/kubelet-wrapper.md

```diff
@@ -0,0 +1,46 @@
+# Kubelet Wrapper Script
+
+The kubelet is the orchestrator of containers on each host in the Kubernetes cluster — it starts and stops containers, configures pod mounts, and other low-level, essential tasks. In order to accomplish these tasks, the kubelet requires special permissions on the host.
+
+CoreOS recommends running the kubelet using the rkt container engine, because it has the correct set of features to enable these special permissions, while taking advantage of all that container packaging has to offer: image discovery, signing/verification, and simplified management.
+
+CoreOS ships a wrapper script, `/usr/lib/coreos/kubelet-wrapper`, which makes it very easy to run the kubelet under rkt. This script accomplishes two things:
+
+1. Future releases of CoreOS can tweak the system-related parameters of the kubelet, such as mounting in /etc/ssl/certs.
+1. Allows user-specified flags and the desired version of the kubelet to be passed to rkt. This gives each cluster admin control to enable newer API features and easily tweak settings, independent of CoreOS releases.
```

Comment:
Huh, I knew about this behavior, but didn't though anyone uses it.

One of the features of Markdown is that it's source is very readable: source file is formatted almost in the same way, as generated HTML, so usually numbering is done in the same way as it expected to be in HTML.

But it's your choice. Sorry for the noise!